store/store.go

```diff
@@ -561,6 +575,10 @@ func (s *Store) newRequest(reqOptions *requestOptions, user *auth.UserState) (*h
 	req.Header.Set("X-Ubuntu-Series", s.series)
 	req.Header.Set("X-Ubuntu-Wire-Protocol", UbuntuCoreWireProtocol)
 
+	if os.Getenv("SNAPPY_DELTA_DOWNLOAD_FORMATS") != "" {
```

Comment:
please compare against the string you want to use to enable the feature, not against the empty string.
This prevents people setting `SNAPPY_DELTA_DOWNLOAD_FORMATS=0` and getting surprised by the feature being enabled.
If you want to be flexible, use something like `if ok, _ := strconv.ParseBool(os.Getenv("...")); ok { ... }`